imodels/rule_set/rule_fit.py

```diff
@@ -389,9 +279,19 @@ def _fit_tree_ensemble(self, X, y):
             self.estimators_ = [[x] for x in self.tree_generator.estimators_]
         else:
             self.estimators_ = self.tree_generator.estimators_
-
+    
     def _extract_rules(self):
```

Comment:
Is this needed when compared to skope? Should it be in convert or is it too specific to rulefit?

Reply:
RuleEnsemble only allowed unique antecedents and sorted the rules by prediction value, which skope doesn't—I'm not sure if this is actually part of RuleFit or if it's just something the RuleFit implementers thought would be a good idea. But it seems like this might be redundant for skope bc it dedups later on

Reply:
okay cool - sounds good how it is then